samples/Tests/Nested.ColumnSet.Bleed.Complex.json

```diff
@@ -0,0 +1,154 @@
+{
+    "type": "AdaptiveCard",
+    "body": [
+        {
+            "type": "TextBlock",
+            "text": "Only inner left and right most containers bleed to edges.\nNotice that column with Y has longer width",
+            "wrap": true
```

Comment:
json files should be indented with tabs. have you [configured your editor](https://editorconfig.org/#download) to apply our formatting?

Reply:
it's configured on wsl. i might have to update it on mac

---
In reply to: [268269359](https://github.com/Microsoft/AdaptiveCards/pull/2551#discussion_r268269359) [](ancestors = 268269359)